pkg/sources/gitlab/gitlab.go

```diff
@@ -53,9 +52,11 @@ type Source struct {
 }
 
 // Ensure the Source satisfies the interfaces at compile time.
-var _ sources.Source = (*Source)(nil)
-var _ sources.SourceUnitUnmarshaller = (*Source)(nil)
-var _ sources.Validator = (*Source)(nil)
+var _ interface {
```

Comment:
Did we decide to use this version of the compile check? I know there was some convo out of band around errors not being as clear, not sure if that was a completely separate convo or related to this?

Reply:
that was me asking your question internally 😄

Reply:
This PR probably prompted the convo - I'll change it back